python/src/main/python/drivers/runspv.py

```diff
@@ -220,28 +257,26 @@ def prepare_shader(output_dir: str, shader: Optional[str]):
 TIMEOUT_APP = 30
 
 
-def adb_helper(adb_args: List[str], check, stdout: Union[None, int, IO[Any]]):
+def adb_helper(adb_args: List[str], check, stdout: Union[None, int, IO[Any]]=subprocess.PIPE):
```

Comment:
Maybe add a verbose argument here that can be passed to subprocess_helper() ?

Reply:
Sounds good. Done. We may also want to actually make some calls verbose, but it seems to look OK from some quick testing.